wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -288,6 +288,24 @@ def run(
                 f"{LOG_PREFIX}Note: no resource args specified. Add a Kubernetes yaml spec or other options in a json file with --resource-args <json>."
             )
         _logger.info(f"Running Kubernetes job with resource args: {resource_args}")
+
+        if (
+            "wandbConfigVersion" not in resource_args
```

Comment:
Can we make this `_wandbConfigVersion`

Reply:
Discussed offline, will make this change with the next PR

Reply:
Discussed (again) - Going to delete this PR and not support versioning at all since backwards compatibility isn't a concern right now.